client/sites/overview/components/migration-overview/test/index.tsx

```diff
@@ -121,9 +121,14 @@ describe( 'MigrationOverview', () => {
 		it( 'shows the migrating started instructions', () => {
 			const site = buildMigrationSite( { status: 'started', how: 'difm' } );
 
-			render( <MigrationOverview site={ site } /> );
+			const { getByText } = render( <MigrationOverview site={ site } /> );
 
-			expect( screen.queryByText( /Your migration is underway/ ) ).toBeVisible();
+			expect( getByText( /We've received your migration request/ ) ).toBeVisible();
+			expect(
+				getByText(
+					/Our team has received your shared details and will review your site to make sure we have everything we need./
```

Comment:
Suggested update:

> Our team has received your details. We will review your site to make sure we have everything we need.

Reply:
As I started writing that edited comment, I made another minor tweak to help with readability. I removed the and.